README.md

```diff
@@ -20,7 +20,6 @@
 Welcome to OpenNLP Site Source Code
 ====================================
 
-[![Build Status](https://api.travis-ci.org/apache/opennlp-site.svg?branch=master)](https://travis-ci.org/apache/opennlp-site)
```

Comment:
Should we replace this by the GH action badgw? (Gine by me either way).

Thanks!

Reply:
Good catch / idea. I added the GH badge + the missing license header for the gh actions file.